import "mapreduce/yt/interface/protos/extension.proto";

message TLoginRecord
{
    optional string Name = 1 [(NYT.column_name) = "name"];
    optional string Login = 2 [(NYT.column_name) = "login"];
    optional int64 Uid = 3 [(NYT.column_name) = "uid"];
}

message TNameUidRecord
{
    optional string Name = 1 [(NYT.column_name) = "name"];
    optional int64 Uid = 2 [(NYT.column_name) = "uid"];
}

message TNameCountRecord
{
    optional string Name = 1 [(NYT.column_name) = "name"];
    optional int64 Count = 2 [(NYT.column_name) = "count"];
}
